README.md

```diff
@@ -93,6 +93,7 @@ Table of contents:
 * [Bit](#angular-2-components-with-bit)
 * [Security](#security)
 * [NgRx](#ngrx)
+* [Permission](#permission)
```

Comment:
This is unnecessary. If you have a component lib or something similar, I'd place it under [Components](https://github.com/AngularClass/awesome-angular#components-1).